src/io/flutter/dart/DartAnalysisServerServiceEx.java

```diff
@@ -35,7 +36,12 @@
   public DartAnalysisServerServiceEx(DartAnalysisServerService base, AnalysisServer analysisServer) {
     this.base = base;
     this.analysisServer = analysisServer;
-    this.requestSink = ReflectionUtil.getField(analysisServer.getClass(), analysisServer, RequestSink.class, "requestSink");
+
+    RequestSink requestSink = ReflectionUtil.getField(analysisServer.getClass(), analysisServer, RequestSink.class, "requestSink");
```

Comment:
I assume this use of reflection will go away with upstream changes to the Dart plugin? If so, can you add a TODO: / note here?